file-formats/ront/examples/z_aff_example_ront.ront.json

```diff
@@ -0,0 +1,34 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example description for SAP Object Type.",
+    "originalLanguage": "en"
+  },
+  "typeCategory": "businessObject",
+  "rootNodeType": {
+    "nodeName": "Z_AFF_EXAMPLE_RONT",
+    "isRoot": true,
+    "nodeNameRaw": "rootNodeType",
+    "description": "Short text description for root SAP Object Node Type."
+  },
+  "nodeTypes": [
+    {
+      "nodeName": "NODETYPE1",
+      "isRoot": false,
+      "nodeNameRaw": "nodeType1",
```

Comment:
As far as I understood `nodeName` and `nodeNameRaw` can have completely different content (like in the root node), don't they. Otherwise (if the conenten is the same and differ only in the case) one can discuss whether the `rawNodeName` would be sufficient, because from this we can "calculate" the `nodeName`. Developers would have to specify it only once.

Reply:
Very good point. After discussing this question, we decided to enforce that the nodeNameRaw converted upper case has to be equal to the node_name. There will be a validation ensuring this.  
Making this requirement, the node_name can now be calculated from node_name_raw, so that it no longer needs to be part of the AFF interface.